packages/oasis-actions/src/operations/aave/v3/open.ts

```diff
@@ -85,78 +81,82 @@ export const operationDefinition = {
 }
 
 export async function open({
+  collateral,
+  debt,
   deposit,
-  swapArgs,
-  addresses,
-  flashloanAmount,
-  borrowAmountInBaseUnit,
-  collateralTokenAddress,
-  debtTokenAddress,
-  eModeCategoryId,
+  swap,
+  flashloan,
   proxy,
-  user,
-  isDPMProxy,
-  positionType,
+  position,
+  emode,
+  addresses,
 }: OpenArgs): Promise<IOperation> {
+  const depositAmount = deposit?.amount || ZERO
+  const depositAddress = deposit?.address || NULL_ADDRESS
+
+  const isDepositingCollateral = deposit?.address === collateral.address
```

Comment:
You can use `depositAmount` here which has been already checked on `line:94`.

Reply:
I don't follow this to be honest.  On line:95 you have `const depositAmount = deposit?.address || NULL_ADDRESS`.
Why can't you use `depositAddress` on line ` const isDepositingCollateral = depositAddress === collateral.address`?